### ISSUE TITLE
Castle.Facilities.AspNet.WebApi not on Nuget

### ISSUE THREAD

**Issue by pbehensky** [castleproject/Windsor#427]
According to information in https://github.com/castleproject/Windsor/blob/master/docs/aspnetwebapi-facility.md there should be Castle.Facilities.AspNet.WebApi package on Nuget, but it looks like there is no such package.

**Comment by fir3pho3nixx** [comment]
Yes, we are still in review and trying help some folks out before we release v5. This will be a major step, so we want to make sure we get this right. Not having internet at home for over a week has not helped :/

**Comment by pbehensky** [comment]
Thanks. I didn't notice that it is not released yet. Now it makes sense.
